pkg/ibmvpc/parser.go

```diff
@@ -506,6 +531,25 @@ func getVPCAddressPrefixes(vpc *datamodel.VPC) (res []string) {
 	return res
 }
 
+func newVPC(name, uid, region string, ap []string, regionToStructMap map[string]*Region) *VPC {
+	vpcNodeSet := &VPC{
+		VPCResource: vpcmodel.VPCResource{
+			ResourceName: name,
+			ResourceUID:  uid,
+			ResourceType: ResourceTypeVPC,
+			Region:       region,
+		},
+		nodes:           []vpcmodel.Node{},
+		zones:           map[string]*Zone{},
+		addressPrefixes: ap,
+		region:          getRegionByName(region, regionToStructMap),
+	}
+
+	vpcNodeSet.addressPrefixesIPBlock, _ = ipblock.FromCidrList(ap)
```

Comment:
We don't ignore errors in building IPBlocks in the other ctors. Why here?

Reply:
right, forgot this one.. added error handling there